iwyu_lexer_utils.h

```diff
@@ -25,6 +25,12 @@ namespace include_what_you_use {
 using std::string;
 using std::vector;
 
+
+// For a particular source line that source_location points to,
+// returns true if the given text occurs on the line.
+// (Case sensitive.)
+bool LineHasText(clang::SourceLocation source_location, clang::StringRef text);
```

Comment:
D'oh. My search for clang::StringRef missed this because the generated Visual Studio projects don't include the header files, only cc files.